### PR TITLE
Delete icon

### PR DESCRIPTION
**Pull request by JuliusMalisauskasDB**
Delete button

### CODE REVIEW

File: src/DeleteButton.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    theme: ThemePropType
+  };
+
+  static propTypes = {
+    testID: PropTypes.string.isRequired,
+    onTap: PropTypes.func
+  };
+
+  render() {
+    const containerStyle = StyleSheet.create({
```
Comment (remanation) [inline src/DeleteButton.js]: When styles are set dynamic I don't think there are any benefit in calling StyleSheet.create

File: src/InputPassword.js [hunk 2]
```diff
@@ -1,40 +1,65 @@
...
 import InputPasswordChar from './InputPasswordChar';
+import { DeleteButton } from './DeleteButton';
 import { range } from 'lodash';
 
 export class InputPassword extends Component {
   static propTypes = {
     value: PropTypes.string,
     length: PropTypes.number,
-    renderIncorrect: PropTypes.bool
+    renderIncorrect: PropTypes.bool,
```
Comment (remanation) [inline src/InputPassword.js]: Maybe the property should just be isIncorrect or maybe isValid is better. When a property is called someting with render... I think is requires a function for handeling some rendering :-)

File: src/InputPassword.js [hunk 3]
```diff
@@ -1,40 +1,65 @@
...
+    onDeleteTap: PropTypes.func
   };
 
   constructor(props) {
     super(props);
   }
 
   render() {
-    const styles = StyleSheet.create({
```
Comment (remanation) [inline src/InputPassword.js]: Think best practise is having StyleSheet.create outside the component and at the end of the file. 
"By putting it at the end of the file, you also ensure that they are only created once for the application and not on every render."

File: src/InputPasswordChar.js [hunk 4]
```diff
@@ -94,28 +94,28 @@ export default class InputPasswordChar extends Component {
 
   getProgressStyles = () => {
     var height = this.state.charAnimated.interpolate({
```
Comment (remanation) [inline src/InputPasswordChar.js]: should this be const instead of var?

File: src/InputText.js [hunk 5]
```diff
@@ -52,13 +53,28 @@ export class InputText extends Component {
 
   render() {
     const styles = StyleSheet.create({
```
Comment (remanation) [inline src/InputText.js]: Should be moved to the end of the file.
"By putting it at the end of the file, you also ensure that they are only created once for the application and not on every render."

File: src/Numpad.js [hunk 6]
```diff
@@ -29,17 +29,19 @@ export class Numpad extends Component {
         left: 0,
         right: 0,
         bottom: 0,
-        height: 280,
-        paddingTop: 20,
-        paddingBottom: 20,
+        height: 300,
```
Comment (remanation) [inline src/Numpad.js]: Move styles to the end of the file

File: src/InputPassword.js (hunk 2 above)
Comment (JuliusMalisauskasDB) [inline src/InputPassword.js]: It was like that in our Html5 app so I left the same name. Yes I can replace it with the better name you suggested.

**Pull request merged**